### DISCUSSION TITLE
[help]

### DISCUSSION THREAD

**Discussion by billdenney** [ropensci/targets#1157] [Help]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

I have a target pipeline where I'm trying to check a lot of files before processing them (probably irrelevant, but giving context just in case).

At the end of a run, I see an error:

```
Last error: no applicable method for 'store_class_format' applied to an object of class "c('NA', 'character')"
```

I can't immediately share the pipeline as it contains proprietary information.  Can you please give any suggestions about how to track down this error (so that I can give a reprex)?

**Comment by wlandau** [comment]
Could be an issue with a custom-defined `tar_format()`, or running a pipeline using current `targets` which was originally created using a much older version of `targets`.

**Comment by wlandau** [comment]
Maybe try avoiding custom `tar_formats()` and see if that fixes it? Might help narrow it down for a reprex.

Closing the discussion since it is a bit old, but let me know if the original issue is still a problem.

**Reply by billdenney** [reply]
I think that the issue had to do with duplicated names like the below info.  But, I can't reproduce it with this.

```
myfun <- function()
tar_plan(
  myfun = 1
)
```
